### ISSUE TITLE
"Prosocial": let's use it and make it the yardstick

### ISSUE THREAD

**Issue by codejack42** [flattenthecurve/guide#136]
**Content to be added or changed:**

A wise woman once said: "There is hope. You’re it. That’s the work." In addition to being absolutely factually correct, it's also deeply and radically prosocial.

I think a corollary is, "As we make our way through this thing, those of us who are more prosocial will on the whole suffer less in trauma, and carry less trauma with us, than those who are more antisocial."

By introducing, using, and juxtaposing the terms "prosocial" and "antisocial", we can without much effort put a simple yet powerful and and effective tool in the reader's hands, and be much more assured of the behavioral outcomes we're encouraging and looking for.

It's safe to say that the whole purpose of "Update and Thorough Guidance" is to encourage maximal prosocial behavior such as checking on your neighbors, and discourage antisocial behavior such as barricading yourself inside, armed to the teeth and hoarding toilet paper. If what we're encouraging and looking for is more prosocial behavior and less antisocial behavior, why not just tell people that straight up, and then tell them again as a consistent thread throughout the guidance?

Right now, what's playing out at the highest levels of US government is a walking, talking (boasting, even) textbook example of Antisocial Personality Disorder. Clearly counterproductive at best, and costing way too many lives and way too much agony. Trauma galore. We don't want or need that.

And it's very, very clear that to the extent that there's going to be prosocial anything directed at helping the populace get through this thing alive and well, it's going to come about in spite of the highest levels of our government, not because of it.

Framing our discussion in terms of roughly "Prosocial, yes! Antisocial, no!" instantly clarifies for the reader what we, and they, are striving for, and striving against. It gives all of us a clear and simple way to view, select, and measure our behaviors in a difficult time when many things may become muddied. It gives all of us a focus point and beacon in a situation where our values can so easily become disoriented (to the point where we even find ourselves fighting over toilet paper).

Can we be up front about that and adopt the term "prosocial" as our watchword, and encourage people to make something like "prosocial to the max" their yardstick for how they decide what to do and how to do it? And can we make the clear distinction between prosocial and antisocial behaviors, and encourage people to do likewise for themselves and others, and to continually default to prosocial rather than antisocial behaviors?

Our culture has more than enough antisocial tendencies and traits, and it's these we'll need to overcome and move past in order to get through this thing with the minimum amount of unavoidable trauma. And what we want to be, and need to be, on the other side of this thing is a kinder, more universally helpful--that is to say, a more prosocially inclined--people.

So why not just use the terms "prosocial" and (to a lesser extent) "antisocial" throughout so that people begin to get accustomed to seeing their own behaviors in that light? It would be a great outcome if two readers were talking, unsure of how to proceed with something, and one of them said, "Well, what's the most prosocial thing we could do? Because I think we should do that."

Plus, there's immediate utility in it as well, helping to frame things in a way that helps people sort their priorities quickly and well. For example:

* People say "People who go out and buy a ton of toilet paper are overreacting." Actually, no, they're not *overreacting*, they're just channeling their very appropriate level of fear into a behavior that's antisocial and not prosocial. And we don't want or need that. Well then, what's the better, more prosocial behavior we all want and need? And how do we help people understand the importance of developing, maintaining, and encouraging more prosocial behaviors, and how do we motivate them in the direction of the more prosocial?

* People say that social distancing  and self-isolating and are antisocial. Heck no, they're not antisocial. For most people at this point, social distancing and self-isolation are the most prosocial behaviors they can possibly adopt. We want and need that. How do we help people understand that difficult choices which may appear to be alienating and weakening are in actuality unifying and strengthening, and confer upon us the special pleasure that is the prosocial?

* People say "Stay the fuck home". If all your employees are bookkeepers who can do all their work from home, yes, telling them stay the fuck at home is highly prosocial (we want and need that!), while telling them to come the fuck in to work is highly antisocial (we don't want or need that.) But if all your employees are respiratory therapists, telling them to stay the fuck home is highly antisocial (and--what? We don't want or need that!), while telling them to come the fuck in to work is prosocial as all getout (and--wait for it--we want and need need that!). But for some people--farm workers and day care workers, for example--it's not so cut and dry. How can we make it easier for them to figure out what's the best choice they can make?

We're trying to guide people into and through a horrible situation with encouragement and expectation that they should, can, and will choose prosocial and not antisocial behaviors. Let's just tell it exactly like it is and give them one big, simple, powerful tool so they can make that happen.

**Comment by jmcmurry** [comment]
Yes, all of this, thanks @codejack42 !

**Comment by jmcmurry** [comment]
I've mostly been really impressed with people's behavior. There are some that are using this excuse to be their worst selves and these bad actors are getting the clicks. You can't reach everyone.

**Comment by codejack42** [comment]
@jmcmurry agree 100%. One end of the curve is the already-highly-prosocial who don't ever need convincing. The other end is the highly-antisocial folks who aren't ever going to be convinced to be less antisocial. For the rest of us middle-dwellers flipping between prosocial and antisocial seems to be a lot more situational and contextual. The big fat middle is who needs the reliable powerful tool and clear yardstick ready to guide them home every step of the way. Help in understanding and evaluating and choosing prosocial behaviors in every situation and context. And convincing people to be prosocial does work. Why, just today Jeff Bezos himself says "Amazon is now dealing only in household staples and medical supplies", and here I've been thinking that Amazon and Jeffy are nothing but the stuff of antisocial legend. Well, somehow they got convinced at least for now, and for now at least I'm willing to believe they've become convinced to do the right thing for the right reason. Peace. Work. Gratitude.

**Comment by pitviper6** [comment]
Agreed! Here in Chicago people have been great (aside from the Saturday pub crawlers) - all my neighbors checking up on each other and the local Aldis is fairly well stocked still. Today was beautiful so lots of people out with their dogs and children but are keeping moving and not clumping in groups. 

Also, I'm one of the very few remote workers at Catalyst, and our daily pet and random channels were on fire today with people posting silly stuff because they can't communicate one on one in the office. It made all of us feel still connected and that we were keeping stuff going. If nothing else, everyone can bond over webex jokes.

**Comment by codejack42** [comment]
Signal Private Messenger on my laptop has become my rockstar international solo and groups comms tool--messaging, vid & voice calls, pics and memes, file exchange, seamless sync with my phone--I feel like it does everything for me short of making me end-to-end encrypted french toast. Unexpectedly (and maybe a little worrisome), I'm developing a legit minor affection and appreciation for Slack.
